src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+```
+</Collapser>
+</CollapserGroup>
+
+### User engagement [#user]
+
+You should measure the value of an incident by the amount of attention it receives. The amount of engagement an individual alert receives is a direct measurement of its value. More engagement implies a valuable alert, while less (or zero) engagement implies an alert may simply be noisy and should be modified or disabled.
+
+There's a significant difference between measuring the moment of incident awareness and acknowledging when resolution activity begins. If you're using an integration with New Relic alerts, be sure that the `Acknowledge` event sent to New Relic triggers when resolution activity begins, not when the incident is sent to the external incident management tool. 
+
+<Callout variant="tip">
+For more information regarding standard incident management processes, see "[Incident management process: 5 steps to effective resolution, posted on August 31, 2020 by OnPage Corporation.](https://www.onpage.com/incident-management-process-5-steps-to-effective-resolution) in reference to [ITIL4](https://itsm.tools/its-here-itil-4-explained)"
+</Callout>
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-pct-ack"
+    title="Percentage acknowledged KPI"
+  >
+This identifies the percentage of incidents that you've engaged with by having their acknowledgement flag set to `true`. Typically you should compare the current and previous weeks.
+
+**Goal:** Increase the percentage of incident engagement.
+
+**Best practices:**
+* Ensure that your DevOps team knows when it's appropriate to acknowledge an incident alert, if applicable.
+* Gamify alert acknowledgement to drive usage.
+* Discourage mass acknowledgement exercises.
+
+```sql
+FROM NrAiIssue SELECT filter(count(*), WHERE event='acknowledge')/filter(count(*), WHERE event='create')*100 AS '% Investigated' WHERE priority='CRITICAL' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-mtti"
+    title="Mean time to investigate (MTTI) KPI"
+  >
+This identifies the average time it takes for you to acknowledge an incident. Typically you should compare the current and previous weeks.
+
+**Goal:** Reduce the mean time to investigate.
+
+**Best practices:**
+* Work at building incident responder's confidence in alerts.
+* Ensure that valuable alerts are acknowledged.
+* Incentivize response teams to respond quickly to alerts.
+
+```sql
+FROM NrAiIssue SELECT average(acknowledgeTime - activateTime) / 60000 AS 'Incident MTTI (minutes)' WHERE event = 'acknowledge' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+</CollapserGroup>
+
+## What's next? [#next]
+
+Once you implement the AQM process from the [previous doc](/docs/tutorial-create-alerts/improve-with-alerts/), you'll see significant reductions in the volume of alerts while maintaining reliability and stability. Your AQM KPIs, which you can measure above, will provide quantifiable proof of these improvements.
```

Comment:
`will provide quantifiable proof of these improvements.` if I read this out loud, I stumble over it.